spec/datagrid/columns_spec.rb

```diff
@@ -32,6 +33,18 @@
       expect(subject.header).to eq(["Shipping date", "Group", "Name", "Access level", "Pet"])
     end
 
+    it "translates column with deprecated namespace" do
+      store_translations(:en, datagrid: {ns45_translated_report: {columns: {name: "Navn"}}}) do
+        expect(Ns45::TranslatedReport.new.header.first).to eq("Navn")
+      end
+    end
+
+    it "translates column with namespace" do
+      store_translations(:en, datagrid: {:"ns45/translated_report" => {columns: {name: "Navn"}}}) do
```

Comment:
What does `ns45/translated_report` mead? Shouldn't it be `ns45.translated_report` for i18n property:

``` yaml
ns45:
  translated_report: 
    columns:
      name: 'Nombre';